main.tf

```diff
@@ -68,7 +68,8 @@ module "service" {
 
   assign_public_ip = var.assign_public_ip
 
-  enable_execute_command = var.enable_execute_command
+  enable_execute_command     = var.enable_execute_command
+  task_placement_constraints = each.value.task_placement_constraints

```

Comment:
@balajimoses is this a separate variable or part of `service_map` ?

Reply:
yea.. shouldn't this be from `var.task_placement_constraints` ?

Reply:
its a separate variables , but its not called anywhere in the main.tf. So i thought to use inside the service map. Please suggest your opinion on this